crates/modelardb_types/src/types.rs

```diff
@@ -145,6 +146,21 @@ impl TimeSeriesTableMetadata {
             ));
         }
 
+        // If the schema contains an unsupported data type that does not correspond to a timestamp,
```

Comment:
Maybe extend the comment to explain how this can happen, I would assume the parser would prevent this?